actions/get_issue.yaml

```diff
@@ -39,3 +39,8 @@ parameters:
     description: True to include linked issues.
     required: true
     default: false
+  sanitize_formatting:
+    type: boolean
+    description: True to remove jinja template artifacts.
```

Comment:
nit: can we reword to something like `When set to true removes jinja template artifacts.`

Reply:
UpdzAted in latest commit.